### PR TITLE
Correct the instructions

### PR DESCRIPTION
**Pull request by sinsukehlab**
### Summary

Just an `_underscore_` makes no sense at all.

### Changes

Correct the instructions by deleting the nonsense and adding a meaningful hint.

Closes:

### Task list

- [x] For workflow changes, I have verified the Actions workflows function as expected.
- [x] For content changes, I have reviewed the [style guide](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Review by rsese — APPROVED** [review]
Looks good to me, thanks!

**Review by Arry-eng — COMMENTED** [review]
Further changes are required as per comments in issue/bug #44

File: .github/steps/2-fix-the-test.md [hunk 1]
```diff
@@ -30,9 +30,6 @@ If the checks don't appear or if the checks are stuck in progress, there's a few
 
 ### :keyboard: Activity: Fix the test
 
-1. Update the code in the `ci` branch to get the test to pass. You need to look something like this:
-  ```markdown
-  _underscore_
-  ```
+1. Update the contents in the `ci` branch to get the test to pass. You need to look at the logs to see what caused the test to fail.
```
Comment (Arry-eng) [inline .github/steps/2-fix-the-test.md:33]: To fix it, open the ci.yml file and go to the last line. Try removing the dot "." in the command "npx remark  **.** --use remark-preset-lint-consistent --frail".

2. Make a change to the README.md file. Make sure the selected branch is "ci".
3. Commit your change to this branch.

**Review by Arry-eng — COMMENTED** [review]
This change is necessitated by the bug #44 . Please refer to the comments by @Arry-eng for details.

**Pull request merged**